puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/foreman_infra.groovy

```diff
@@ -80,3 +85,16 @@ def runCicoJobsInParallel(jobs) {
         }
     }
 }
+
+def notifyDiscourse(heading=null) {
+    body = [heading, env.BUILD_URL]
+    if (currentBuild.description) {
```

Comment:
What is currentBuild? I think that should be passed in. Keep this as a simple function.